docs/source/release_notes/release-notes.rst

```diff
@@ -83,6 +83,17 @@ develop
          - Reduced logging noise from large Cassandra gets and puts by removing ERROR messages and only providing stacktraces at DEBUG.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1590>`__)
 
+    *    - |new|
+         - Upon startup of a Cassandra-backed AtlasDB client with a
```

Comment:
nit: rather than 120 chars per line, we have one sentence per line in docs (including release notes). Exceptionally long sentences can be split at commas etc.